templates/page/page--ds.tpl.php

```diff
@@ -186,48 +186,6 @@
   </div>
 </section>
 
-<footer class="footer">
-<?php if (!empty($page['footer_improved'])): ?>
-  <section class="footer__improved">
-    <div class="container-fluid">
-      <?php print render($page['footer_improved']); ?>
-    </div>
-  </section>
-<?php endif; ?>
-
-  <div class="footer__top">
-    <div class="container-fluid">
-      <div class="row">
-      <?php if (!empty($page['footer_left'])): ?>
-        <div class="footer__column">
-          <?php print render($page['footer_left']); ?>
-        </div>
-      <?php endif; ?>
-
-      <?php if (!empty($page['footer_middle'])): ?>
-        <div class="footer__column">
-          <?php print render($page['footer_middle']); ?>
-        </div>
-      <?php endif; ?>
-    
-      <?php if (!empty($page['footer_right'])): ?>
-        <div class="footer__column">
-          <?php print render($page['footer_right']); ?>
-        </div>
-      <?php endif; ?>
-      </div>
-    </div>
-  </div>
-
-  <div class="footer__bottom">
-    <div class="container-fluid">
-      <div class="row">
-        <div class="col-sm-12">
-        <?php if (!empty($page['footer_bottom'])): ?>
-          <?php print render($page['footer_bottom']); ?>
-        <?php endif; ?>
-        </div>
-      </div>
-    </div>
-  </div>
+<footer class="ecl-footer">
```

Comment:
I have a question regarding this... and to be honest, I don't know how to deal with the solution.
Should we include the `<footer>` tag inside the footer template or we should let it live in the page template ?